modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaGetFileMethodCheck.testjava

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaGetFileMethodCheck {
+
+	public void method() {
+
+    	// Test local service class method
+
+		File file = dlFileEntryLocalService.getFile(fileEntryId, fileEntryVersion, true);
+
+		// Test utility class method
+
+		return DLFileEntryLocalServiceUtil.getFile(fileEntryId, fileEntryVersion, true);
+
+    }
+
+	@Reference
+	private DLFileEntryLocalService dlFileEntryLocalService;
+
```

Comment:
@TamyrisTorres, please make sure the indentation is correct, more details about it can be found in https://liferay.atlassian.net/wiki/spaces/ECU/pages/2305327471/How+to+create+a+code+upgrade+automation#Code-Standards :wink: